db/migrate/20250218113307_add_flexi_working_details_provided_to_vacancies.rb

```diff
@@ -0,0 +1,7 @@
+class AddFlexiWorkingDetailsProvidedToVacancies < ActiveRecord::Migration[7.2]
+  # rubocop:disable Rails/ThreeStateBooleanColumn
```

Comment:
I have allowed 3 states here as I don't think we can really backfill `flexi_working_details_provided` for existing vacancies. I thought about creating a migration to set it to false if flexi_working was empty but seems like it kind of defeats the point of the ticket as those that have 'n/a' or similar here would have flexi_working_details_provided set to true, which I think is exactly what we are trying to avoid.

Reply:
maybe we should have a comment saying that this has to exist because of the modelling, as the vacancy model stores part-created vacancies (where this field and its ilk can be unknown values)